int32 id
RoadLinePoint[] points
